init.lua

```diff
@@ -110,7 +118,7 @@ local function entity_pos(obj, offset)
 					obj:get_pos()
 				),
 				vector.multiply(
-					obj:get_player_velocity(),
+					velocity,
```

Comment:
`velocity` must be a vector (table).  When `get_*velocity` returns `nil`, it means that the object is no longer valid (i.e. deleted).

Reply:
Oh you're right, I don't know how I didn't notice that, it hasn't used the fallback values yet though so the condition is working, I'll fix now

Reply:
@SmallJoker  that went past me, fluxionary and luk3yx and none of us caught it :facepalm: should be good now

Reply:
(( honestly I hate this entire section of code, it should just be `math.round(obj:pos())`, I don't know why we need to project velocity, but I suppose on a server it's better than lagging behind ))

Reply:
To simplify things, you could let `velocity` be `nil` on error and change this line to
`velocity or { x=0, y=0, z=0 }` which makes the fallbacks above superfluous.

Reply:
@SmallJoker That's changed now, much cleaner, thanks